xm-commons-timeline/src/main/java/com/icthh/xm/commons/timeline/TimelineEventProducer.java

```diff
@@ -185,17 +187,17 @@ private static Object getHeaderValue(HttpServletRequest request, String name) {
     }
 
     private static Object getEntityField(String entity, String field) {
-        try {
-            return JsonPath.read(entity, "$." + field);
-        } catch (Exception ex) {
-            log.trace("JsonPath exception", ex);
+        List<String> prefixes = asList("$.", "$.xmEntity.", "$.data.");
```

Comment:
could we have more explicit way to define which container to use instead of trying to fine right prefix in cycle?
For example add prefix as additional parameter for method.

Reply:
справа в тому шо идея була сделать хендлер на любое апи, и не хочеться менять апи под этот хендлер, а id может лежать в 3 местах, и я просто изменил цепочку вложеных try catch на цикл, не меняя логику